### PR TITLE
[MXNet,Pytorch][EIA]Skip the p3dn test in EIA

### PR DESCRIPTION
**Pull request by TusharKanekiDey**
*Issue #, if available:*

## Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [build] | [test] | [build, test] | [ec2, ecs, eks, sagemaker]

#### EIA Checklist
* When creating a PR:
- [x] I've modified ```src/config/build_config.py``` in my PR branch by setting ```ENABLE_EI_MODE = True```
* When PR is reviewed and ready to be merged:
- [x] I've reverted the code change on the config file mentioned above

* For reviewer, before merging, please cross-check:
- [x] I've verified the code change on the config file mentioned above has already been reverted

*Description:*
Trying to skip the p3dn test for PT and MXNet EIA

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by arjkesh** [comment]
@TusharKanekiDey can you add back the EI related information to the checklist, since this is an EI related PR?

File: test/test_utils/ec2.py [hunk 1]
```diff
@@ -37,8 +37,7 @@ def get_ec2_instance_type(default, processor, disable_p3dn=False):
             "Default instance type should never be p3dn.24xlarge"
         )
     instance_type = os.getenv(f"EC2_{processor.upper()}_INSTANCE_TYPE", default)
-    #Not running eia test on p3dn instances
-    if instance_type == p3dn and (disable_p3dn or processor == "eia"):
+    if instance_type == p3dn and disable_p3dn:
```
Comment (arjkesh) [inline test/test_utils/ec2.py:40]: is this function used for eia at all? it probably can be instead of get_ec2_accelerator type

Comment (TusharKanekiDey) [reply test/test_utils/ec2.py:40]: No, This function is used for CPU and GPU processor type.
To Decouple them, we kept get_ec2_accelerator type to fetch the EI accelerator type.
I can combine both of them if needed.

**Comment by TusharKanekiDey** [comment]
> @TusharKanekiDey can you add back the EI related information to the checklist, since this is an EI related PR?

Done, I was planning to add them back when they are ready for review.
Thanks for pointing that out.

File: test/test_utils/ec2.py (hunk 1 above)
Comment (arjkesh) [reply test/test_utils/ec2.py:40]: no need to combine in this PR, but probably we should remove "eia" from allowed_processors in line 28 if we are specifically decoupling eia from this function

Comment (TusharKanekiDey) [reply test/test_utils/ec2.py:40]: Sure, I will remove eia from allowed_processors in  line 28.

**Pull request merged**